_posts/en/newsletters/2019-05-21-newsletter.md

```diff
@@ -0,0 +1,221 @@
+
+## Bech32 sending support
+
+*Week 10 of 24 in a [series][bech32 easy] about allowing the the people
+you pay to access all of segwit's benefits.*
+
+{% comment %}<!-- weekly reminder for harding: check Bech32 Adoption
+wiki page for changes -->{% endcomment %}
+
+Up until this point in our series encouraging wallets and services to
+support sending to bech32 native segwit addresses, we've focused
+almost exclusively on technical information.  Today, this section
+expresses an opinion: the longer you delay implementing bech32 sending
+support, the worse some of your users and potential users will think of
+your software or service.
+
+> "They can only pay legacy addresses."<br>
+> "Oh.  Let's look for another service that supports current technology."
+
+Services that only support legacy addresses are likely to become a cue
+to users that minimal development effort is being put into maintaining
+their Bitcoin integration.  We expect that it'll send the same signal to
+users as a website in 2019 that's covered in Shockwave/Adobe Flash
+elements and that claims it's best viewed in Internet Explorer 7 (or
+see an even more [imaginative comparison][nullc bank analogy] written by Gregory
+Maxwell.)
+
+Bech32 sending is not some experimental new technology that still needs
+testing---native segwit unspent outputs currently hold [over 200,000
+bitcoins][].  Bech32 sending is also something that's easy to implement
+(see Newsletters [#38][news38 bech32] and [#40][news40 bech32]).  Most
+importantly, as more and more wallets and services upgrade to bech32
+*receiving* by default, it's going to become obvious which other
+services haven fallen behind by not providing sending support.
+
+If you haven't implemented bech32 sending support yet, we suggest you
+try to get it implemented by 24 August 2019 (the two-year anniversary of
+segwit activation).  Not long after that, Bitcoin Core's next release is
+expected to begin defaulting to bech32 receiving addresses in its GUI
+and perhaps also its API methods (see Newsletters [#40][Newsletter #40]
+and [#42][Newsletter #42]).  We expect other wallets to do the
+same---except for the ones that have already made bech32 their default
+(or even their only supported address format).
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], and [Bitcoin Improvement Proposals
+(BIPs)][bips repo].*
+
+- [Bitcoin Core #15006][] extends the `createwallet` RPC with a new
+  `passphrase` parameter that allows creating a new wallet that's
+  encrypted by default.  Existing wallets can still be converted to
+  encrypted with the `encryptwallet` RPC.
+
+- [Bitcoin Core #15870][] will cause the `importwallet`,
+  `importpubkey`, `importaddress`, and `importprivkey` RPCs to fail if
+  any blocks have been pruned.  Previously, it failed if pruning was
```

Comment:
I think in this case it'd be clearer to first say what the current behaviour is, and then what the change is. For example:

[Bitcoin Core #15870][] makes `importwallet`, `importpubkey`, `importaddress`, and `importprivkey` RPCs succeed more often for pruned nodes.  Previously, those calls would always fail on nodes which were configured to prune the blockchain. After this change, they will only fail if blocks have already been pruned. Pruning can be configured for manual operation...